src/main/webapp/script.js

```diff
@@ -19,75 +19,69 @@ google.charts.load('current', {
 });
 
 function passQuery() {
-  const information = document.getElementById('more-info');
-  information.innerText = 'Please wait. Loading...';
+  document.getElementById('result').style.display = 'block';
   const query = new FormData(document.getElementById('query-form'));
   const personType = query.get('person-type');
-  const action = query.get('action');
   const location = query.get('location');
 
-  const fetchUrl = '/query?person-type=' + personType +
-      '&action=' + action +
-      '&location=' + location;
-  
-  fetch(fetchUrl)
-    .then((response) => {
-      if (response.ok) {
-        response.json().then((jsonResponse) => JSON.parse(jsonResponse))
-        .then((data) => {
-          // data is a 2D array, where the first row is a header row and all
-          // subsequent rows are one piece of data (e.g. for a state or county)
-          information.innerText = '';
-          displayVisualization(data);
-        });
-      } else {
-        console.log(
-          `An error occurred: ${response.status}: ${response.statusText}`);
-      }
-    });
+  fetch('/query?person-type=' + personType + '&location=' + location)
```

Comment:
Oh wait. It looks like you weren't up to date with master -- this is a revert to before I added more dropdown options.